resources/licensing-info.txt

```diff
@@ -27,6 +27,7 @@ Chris "Goober" Parsons:
 * HUD textures for the big mode
 * symbol glyphs in ttf fonts
 * launcher header
+* texture and mesh for gibs
```

Comment:
if mesh was made from V. meshes I don't we should pin authorship to you? If mesh or texture was made fully by you (from scratch) it's okay

Reply:
Good point. I will make the needed adjustments when I submit the next commit to this PR.

The mesh file was created by me but uses geometry created by Volition - this will also not be relevant once the approach is switched to throwing out multiple chunks instead of using this one file.

The texture was created new by me because Volition's original tga as far as I know has been lost (I've never been able to dig it up anyway, and I've looked everywhere I could think to). It does incorporate some Volition-created gore decals to add more detail to it, but the texture is new.